Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.16.3-alpine
```

Comment:
We should really author a Docker compose file to simplify setting up the database as well

Reply:
I agree, to be honest I only wrote the dockerfile for testing it out on my own machine